src/python/CRABClient/JobType/UserTarball.py

```diff
@@ -140,13 +142,13 @@ def upload(self, filecacheurl=None):
 
         self.close()
         archiveName = self.tarfile.name
-        archiveSizeKB = os.path.getsize(archiveName)/1024
+        archiveSizeKB = os.path.getsize(archiveName)//1024
         if archiveSizeKB <= 512 :
           archiveSize = "%d KB" % archiveSizeKB
         elif archiveSizeKB < 1024*10 :
           archiveSize = "%3f.1 MB" % (archiveSizeKB/1024.)
```

Comment:
Another very minor thing, the dot after 1024 is no longer necessary for this division to give a floating point result, could be confusing for someone in the future.